inlong-sdk/dataproxy-sdk-twins/dataproxy-sdk-python/inlong_dataproxy.cpp

```diff
@@ -0,0 +1,57 @@
+/**
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+#include <pybind11/pybind11.h>
+#include <pybind11/stl.h>
+#include <inlong_api.h>
+
+namespace py = pybind11;
+using namespace inlong;
+
+class PyInLongApi : public InLongApi {
```

Comment:
It is better to use the underscore style for naming consistently, without using the camel case naming style.

Reply:
The class name(`InLongApi`) here needs to be the same as the class name defined in the C++ SDK, otherwise pybind won't be able to achieve the correct binding, so the class name here uses camel case naming style to be consistent with the naming in the C++ SDK code.